garak/generators/nvcf.py

```diff
@@ -64,9 +65,11 @@ def __init__(self, name=None, generations=10):
         ),
         max_value=70,
     )
-    def _call_model(self, prompt: str, generations_this_call: int = 1) -> str:
+    def _call_model(
+        self, prompt: str, generations_this_call: int = 1
+    ) -> List[Union[str, None]]:
         if prompt == "":
-            return ""
+            return [None]
```

Comment:
This is an interesting, in https://github.com/leondz/garak/pull/682#discussion_r1608125584 there is suggestion that `None` should be returned when an error is handled. In this case the method just preempts the sending the prompt when it is an empty string. Does the `NVCF` spec document an empty prompt as being an error?

Reply:
Not last time I checked, but it does raise one. I can see where this should go - `try`/`except` first.